### ISSUE TITLE
Let's update and organize

### ISSUE THREAD

**Issue by voytek** [voytekresearch/tutorials#1]
Thius tutorial page needs to be updated, with pointers to the various project-specific tutorials pages where appropriate, but also in a logical order.

The logical should roughly flow thusly:
1) Loading in raw electrophysiological brain data.
2) Referencing, and effect of reference choice on data.
3) Filtering introduction, and how filtering noise (white, pink, etc.) will return an oscillation even where there is none.
4) FFT, and Welch's method for the power spectrum.
5) Spectral CV.
6) Identifying oscillations in the PSD.
7) Time-domain methods for checking for an oscillation.
8)  Cycle-by-cycle waveform analysis.
9) How non-sinusoidal features manifest in the PSD (harmonics, slope, etc.).
10) Where might 1/f come from, and how do we best measure it?

This should also point to our neurodsp and fooof tutorials, where appropriate, so things aren't so scattered and to give a logical flow to our overall process.

**Comment by TomDonoghue** [comment]
Just a note (update for @voytek): we met yesterday (Sydney, Julio, Michael and I) to sketch out a plan for this. To try and keep things organized we plotted out an organization by which this repo has all the background stuff, and then FOOOF / NeuroDSP specific things stay in their own repos (are linked to from here). 

To update for this plan, we sketched out the topics for 9 notebooks that would make up this repo. Sydney, Michael and Julio are each going to re-organize the existing notebooks / materials into these 9 new notebooks, that would then direct you to further tutorials in FOOOF / NeuroDSP, as appropriate. As a first pass, this will just be re-organizing the notebooks that are currently there, and then we can all check in on if there are any missing sections, that need new things writing.

This re-org will be pushed to branches as things come together - so let us all know if this plan sounds good / you have any suggestions, etc.

**Comment by voytek** [comment]
I agree that package-specific tutorials should remain as a tutorial within the package, with this repo pointing to those. That's what I had in mind. Thanks for organizing!
